assets/package.json

```diff
@@ -18,7 +18,7 @@
     "phoenix": "../deps/phoenix",
     "phoenix_html": "file:../deps/phoenix_html",
     "phoenix_live_view": "file:../deps/phoenix_live_view",
-    "uplot": "^1.0.8"
+    "uplot": "github:leeoniya/uPlot#6a8be6c"
```

Comment:
very brave, might as well test out https://github.com/leeoniya/uPlot/commit/31bbf0e8938e2c8e276b2c195beea95131531263 :D

Reply:
i mainly wanted to test out the zoom changes :)  i think, for the purposes of this PR, we're probably safe to go back to 1.0.8 and then upgrade once you cut a proper release.